pygsti/tools/optools.py

```diff
@@ -430,6 +433,13 @@ def entanglement_fidelity(a, b, mx_basis='pp', is_tp=None, is_unitary=None):
     -------
     float
     """
+    # Attempt to cast to dense array. If this is already an array, the AttributeError
+    # will be suppressed.
+    with _contextlib.suppress(AttributeError):
```

Comment:
Can we change the contextlib.suppress calls to an isinstance(_, LinearOperator) instead?

It's more pyGSTi style to check for the base class that provides the used API than a blanket error suppression.

Reply:
Of course! I'm more used to EAFP-style programming, but happy to accomodate LBYL, too :)  Just pushed the changes!